lib/tasks/emotion_selection_notification.rake

```diff
@@ -1,6 +1,9 @@
 namespace :notifications do
   desc 'Send emotion selection email'
   task emotion_selection_invite_all: :environment do
-    EmotionSelectionNotificationWorker.new.run_notification
+    date = Time.zone.now
+    if date.strftime("%A").downcase == ENV["DAY_TO_SEND_INVITES"].downcase
```

Comment:
`date = Date.current` would be preferable to a Time object

If you're going to put this logic into the rake task then you'll need to add testing coverage to the rake task to make sure it works (it will be a very bad user experience it emails went out every day or not at all). It would be easier I think to place it into the `EmotionSelectionNotificationWorker`. Maybe have 2 methods, `run_notification` which does the check and then `run_notification!` which sends emails without running the check.